lib/src/notification/notification.component.ts

```diff
@@ -0,0 +1,176 @@
+import { OnChanges, Component, ViewEncapsulation, Input, Output, EventEmitter, OnInit, SimpleChanges, OnDestroy } from "@angular/core";
+
+export type NotificationStyle = "slide-in" | "bar";
+export type NotificationPosition = "bottom-left" | "bottom-right" | "top-left" | "top-right" | "top" | "bottom";
+export type NotificationTheme = "purple" | "primary" | "danger" | "success" | "warning" | "inverted";
+
+export interface NotificationAction {
+    text: string;
+    action: () => void;
+}
+
+@Component({
+    selector: "sebng-notification",
+    templateUrl: "./notification.component.html",
+    styleUrls: ["./notification.component.scss"],
+    encapsulation: ViewEncapsulation.None,
+})
+export class NotificationComponent implements OnChanges, OnInit, OnDestroy {
+    @Input() className?: string;
+    @Input() dismissable?: boolean;
+    @Input() dismissTimeout?: number;
+    @Input() message?: string;
```

Comment:
the message should not be a string, its too straight forward, notifications can contains links or phone number or an email to click on, you should give to the user the possibility to put whatever he wants